appointment-svc/models/init_app_exc_receiver.ts

```diff
@@ -1,12 +1,12 @@
-const amqp = require('amqplib/callback_api')
-const appActionEmitter = require('../utils/appointmentActionEmitter')
+import amqp from 'amqplib/callback_api'
+import appActionEmitter from '../utils/appointmentActionEmitter'
 require('dotenv').config();
 
 async function receiveMessageFromInitAppExc() {
 
     var router_keys = ['book', 'cancel'];
 
-    amqp.connect(process.env.RABBITMQ_URL, function (error0, connection) {
+    process.env.RABBITMQ_URL && amqp.connect(process.env.RABBITMQ_URL, function (error0, connection) {
```

Comment:
Why do you put the operator '&&' here and in other files?

Reply:
cuz ts wanted me to check for process.env.RABBITMQ_URL being potentially undefined. I guess it is called narrowing in ts

Reply:
Gotcha, also, why are you declaring a variable with 'var'?

Reply:
hz:/

Reply:
all the variables in rabbitmq tutorials defined this way for some reason. Kinda didnt want to change it not to break anything, but I actually did not notice it, so I will look into that later